public/components/overview/office-panel/office-panel.tsx

```diff
@@ -11,55 +11,70 @@
  * Find more information about this on the LICENSE file.
  */
 
-import React, { useState } from 'react';
-
-import { MainPanel } from '../../../components/common/modules/panel';
-import { FilterManager, Filter } from '../../../../../../src/plugins/data/public/';
-//@ts-ignore
-import { getDataPlugin } from '../../../kibana-services';
-import { KbnSearchBar } from '../../kbn-search-bar';
-import { TimeRange, Query } from '../../../../../../src/plugins/data/common';
+import React, { useEffect, useState } from 'react';
+import { MainPanel } from '../../common/modules/panel';
 import { withErrorBoundary } from '../../common/hocs';
-import { MockupTables } from './mockup-tables';
-import { ModuleStats } from './module-stats';
-
-export const OfficePanel = withErrorBoundary(({ ...props }) => {
+import { getErrorOrchestrator } from '../../../react-services/common-services';
+import {
+  UI_ERROR_SEVERITIES,
+  UIErrorLog,
+  UIErrorSeverity,
+  UILogLevel,
+} from '../../../react-services/error-orchestrator/types';
+import { UI_LOGGER_LEVELS } from '../../../../common/constants';
+import { CustomSearchBar } from '../../common/custom-search-bar';
+import { OfficeStats } from './views';
+import { queryConfig } from '../../../react-services/query-config';
+import { ModuleConfig, filtersValues } from './config';
 
-    const KibanaServices = getDataPlugin().query;
-    const filterManager = KibanaServices.filterManager;
-    const timefilter = KibanaServices.timefilter.timefilter;
+export const OfficePanel = withErrorBoundary(() => {
+  const [moduleStatsList, setModuleStatsList] = useState([]);
 
-    const [moduleStatsList, setModuleStatsList] = useState([]);
-    const [filterParams, setFilterParams] = useState({
-        filters: filterManager.getFilters() || [],
-        query: { language: 'kuery', query: '' },
-        time: timefilter.getTime(),
+  /** Get Office 365 Side Panel Module info **/
+  const getModuleConfig = async () => {
```

Comment:
Why is the office Panel the one loading the module config instead of the side panel component that displays this information?

Reply:
To have dummy components as children